File: back-end/src/main/java/com/joaoalencar/projetoimpacta/models/DTOs/ProductDTO.java
package com.joaoalencar.projetoimpacta.models.DTOs;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ProductDTO {

    private int id;

    private String name;

    private String description;

    private float price;

    private int stock;
}

File: api/Domain/DTOs/ProductDTO.cs
namespace api.Domain.DTOs;

public class ProductDTO
{
    public long? Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public decimal Price { get; set; }
    public int Stock { get; set; }
    public long SupplierId { get; set; }
    public IEnumerable<ProductImageDTO>? ProductImages { get; set; }
}

public class ProductImageDTO
{
    public string FileName { get; set; }
    public long ProductId { get; set; }
}
